docs/delegate-resources/rubin-data-academy-2024.rst

```diff
@@ -67,11 +67,11 @@ The goal of this DP0 Rubin Data Academy is to introduce any data-rights holding
 (from undergraduate student to faculty/staff) to both the `DP0.2 dataset <https://dp0-2.lsst.io/data-products-dp0-2/index.html#the-desc-dc2-data-set>`_
 and the `DP0.3 dataset <https://dp0-3.lsst.io/data-products-dp0-3/>`_,
 as well as the RSP, and to provide an accelerated learning experience enabling research on Rubin data with a focus on both DP0 datasets.
-In order to develop a thriving Rubin science community, another key goal of the Virtual Summer School is to provide opportunities
+In order to develop a thriving Rubin science community, another key goal of the Rubin Data Academy is to provide opportunities
 for networking, collaboration, and new-skill acquisition for participants.
 
 **Schedule.**
-The Virtual Summer School will span 4 days: between June 17 - 18 and June 20 - 21, where each day will be split into two 2-3 hr sessions
+The Rubin ata Academy will span 4 days: between June 17 - 18 and June 20 - 21, where each day will be split into two 2-3 hr sessions
```

Comment:
"Rubin ata Academy" --> "Rubin Data Academy"

Reply:
Just fixed the text, and uploaded a modified version of the Agenda.